locale/es.yml

```diff
@@ -1248,113 +1248,113 @@ __TRAIT__: ""
-  else: ""
-  except: ""
-  finally: ""
-  for: ""
-  from: ""
-  global:  ""
-  if: ""
-  import: ""
-  in: ""
-  is: ""
-  lambda: ""
-  nonlocal: ""
-  not: ""
-  or: ""
-  pass: ""
-  raise: ""
-  return: ""
-  try: ""
-  while: ""
-  with: ""
-  yield: ""
+  False: "Falso" # ✅✅✅✅✅✅✅
+  None: "Ninguno" # ✅✅✅✅✅✅✅
+  True: "Verdadero" # ✅✅✅✅✅✅✅
+  and: "Y" # ✅✅✅✅✅✅✅
+  as: "como" # ✅✅✅✅✅✅
+  assert: "afirmar" # ✅❓ What about "asegurar"? ✅ I agree with "afirmar" ✅ ✅ I also agree with "afirmar"✅✅ I agree with "afirmar"
+  async: "asíncrono" # ✅✅✅✅✅✅✅
+  await: "esperar" # ✅✅✅✅✅✅✅
+  break: "interrumpir" # ✅✅✅✅✅Also "romper"✅ Yes, it also could be "romper"
+  class: "clase" # ✅✅✅✅✅✅✅
+  continue: "continuar" # ✅✅✅✅✅✅✅
+  def: "define" # ✅ def it's a short for define, in the traslated version isn't shorted ❓What about using "define" as imperative, because you're defining something? ✅ I agree with "define" ✅✅ I also agree with "define"✅✅
+  del: "eliminar" # ✅✅ same as def ✅ I agree with "eliminar" ✅ could be left as del, but eliminar is good as well✅✅✅Also "borrar"
+  elif: "si no, si" # ✅❓ I added a comma here, it's something like, if this does not satisfy let's try this other ✅ I agree with the comment, it is better having the comma ✅ I agree that having the comma may be beneficial, but it might also be a bit too long (maybe something like no, si is fine, similar to else if)✅✅✅✅
+  else: "si no" # ✅✅✅✅✅✅
+  except: "excepto" # ✅✅✅✅✅✅✅
+  finally: "finalmente" # ✅✅✅✅✅✅✅
+  for: "para" # ✅✅✅✅✅✅
+  from: "desde" # ✅✅✅✅✅✅✅
+  global:  "global" # ✅✅✅✅✅✅✅
+  if: "si" # ✅✅ same in the else keyword, in this context means "si" ✅✅✅✅✅
+  import: "importar" # ✅✅✅✅✅✅✅
+  in: "en" # ✅✅✅✅✅✅✅✅
+  is: "es" # ✅✅✅✅✅✅✅
+  lambda: "función anónima" # there isn't translation for this word to spanish, 🤔 What about using the same word? 🤔 I agree with the second comment, in this case is better to keep the same word "lambda" 🤔 I agree with keeping the word lamba, 'función anónima' may be a bit too much 🤔 I also agree with the second comment 🤔 The word should stay as lambda, since it's a greek word, and in Spanish is the same.🤔 I agree with the second comment 
+  nonlocal: "no local" # ✅✅✅✅✅✅✅
+  not: "no" # ✅✅✅✅✅✅✅
+  or: "o" # ✅✅✅✅✅✅✅
+  pass: "pasar" # ✅✅✅✅✅✅✅
+  raise: "genera excepción" ✅❓ # maybe just "generar" will be ok ✅ # I agree with keeping generar✅❓ In the Python documentation in Spanish is used "lanza" or "levanta"✅
+  return: "regresa" # ✅✅ Regresa sounds more natural for me than retorna ❓I think that in this case "retorna" it is the better way to translate it, as we are talking that a function will run to then return a value, in spanish will be "la función se ejecuta y retorna un valor" ✅ I think regresa would be just fine ❓ The word "retorna" is accepted in the Spanish Python documentation✅
+  try: "intenta" # ✅✅ I also think of using this as imperative ✅ I agree with "intenta" ❓ maybe "tratar" would be better? intenta sounds okay too ✅ "intenta"✅✅
```

Comment:
Both intenta and tratar translate okay for the word try, but personally I like tratar better. It seems to translate better than intenta, maybe new users would look to use tratar instead. This could be changed later on down the line based on user's wants if necessary.